.github/workflows/publish.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       release_created:  ${{ steps.release.outputs.release_created }}
       release_tag:  ${{ steps.release.outputs.tag_name }}
     steps:
-      - uses: google-github-actions/release-please-action@v3
```

Comment:
[`google-github-actions/release-please-action` is archived](https://github.com/google-github-actions/release-please-action).